File: src/main/java/logic/entity/Employee.java
//
// Source code recreated from a .class file by IntelliJ IDEA
// (powered by Fernflower decompiler)
//

package logic.entity;

import java.io.Serializable;
import java.time.LocalDate;
import java.util.ArrayList;

public class Employee implements Serializable{
    private int id;
    private String firstName;
    private String lastName;
    private String patronymic;
    private LocalDate dateOfBirth;
    private String gender;
    private String nationality;
    private String familyStatus;
    private String webSite;
    private String email;
    private String workPlace;
    private Address address = new Address();
    private ArrayList<ContactPhone> phoneList = new ArrayList<ContactPhone>();
    private ArrayList<Attachment> attachmentList = new ArrayList<Attachment>();
    private Photo photo;
    public Employee() {
    }

    public int getId() {
        return this.id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getFirstName() {
        return this.firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return this.lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getPatronymic() {
        return this.patronymic;
    }

    public void setPatronymic(String patronymic) {
        this.patronymic = patronymic;
    }

    public String getGender() {
        return this.gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public LocalDate getDateOfBirth() {
        return this.dateOfBirth;
    }

    public void setDateOfBirth(LocalDate dateOfBirth) {
        this.dateOfBirth = dateOfBirth;
    }

    public String getNationality() {
        return this.nationality;
    }

    public void setNationality(String nationality) {
        this.nationality = nationality;
    }

    public String getFamilyStatus() {
        return this.familyStatus;
    }

    public void setFamilyStatus(String familyStatus) {
        this.familyStatus = familyStatus;
    }

    public String getWebSite() {
        return this.webSite;
    }

    public void setWebSite(String webSite) {
        this.webSite = webSite;
    }

    public String getEmail() {
        return this.email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getWorkPlace() {
        return this.workPlace;
    }

    public void setWorkPlace(String workPlace) {
        this.workPlace = workPlace;
    }

    public Address getAddress() {
        return this.address;
    }

    public void setAddress(Address address) {
        this.address = address;
    }

    public Photo getPhoto() {
        return photo;
    }

    public void setPhoto(Photo photo) {
        this.photo = photo;
    }

    public ArrayList<ContactPhone> getPhoneList() {
        return phoneList;
    }

    public void setPhoneList(ArrayList<ContactPhone> phoneList) {
        this.phoneList = phoneList;
    }

    public ArrayList<Attachment> getAttachmentList() {
        return attachmentList;
    }

    public void setAttachmentList(ArrayList<Attachment> attachmentList) {
        this.attachmentList = attachmentList;
    }

    public String toString() {
        return "Employee{id=" + this.id + ", firstName=\'" + this.firstName + '\'' + ", lastName=\'" + this.lastName + '\'' + ", patronymic=\'" + this.patronymic + '\'' + ", dateOfBirth=" + this.dateOfBirth + ", gender=\'" + this.gender + '\'' + ", nationality=\'" + this.nationality + '\'' + ", familyStatus=\'" + this.familyStatus + '\'' + ", webSite=\'" + this.webSite + '\'' + ", email=\'" + this.email + '\'' + ", workPlace=\'" + this.workPlace + '\'' + ", address=" + this.address + '}';
    }


}

File: src/main/webapp/web/jsp/entity/employee.js
/**
 * Created by aefrd on 26.01.2017.
 */
function Employee() {
    this._id = undefined;
    this._firstName = undefined;
    this._lastName = undefined;
    this._firstName = undefined;
    this._lastName = undefined;
    this._patronymic = undefined;
    this._dateOfBirth = undefined;
    this._gender = undefined;
    this._nationality = undefined;
    this._familyStatus = undefined;
    this._webSite = undefined;
    this._email = undefined;
    this._workPlace = undefined;
    this._address = undefined;
    this._phoneList = [];
    this._attachmentList = [];
    this._photo = [];

}

Employee.prototype = {
    setId : function (id) {
        this._id = id;
    },
    getId : function () {
        return this._id;
    }
};
Employee.prototype = {
    setFirstName : function (firstName) {
        this._firstName = firstName;
    },
    getFirstName : function () {
        return this._firstName;
    }
};
Employee.prototype = {
    setLastName : function (lastName) {
        this._lastName = lastName;
    },
    getLastName : function () {
        return this._lastName;
    }
};
Employee.prototype = {
    setPatronymic: function (patronymic) {
        this._patronymic = patronymic;
    },
    getPatronymic: function () {
        return this._patronymic;
    }
};
Employee.prototype = {
    setDateOfBirth : function (dateOfBirth) {
        this._dateOfBirth = dateOfBirth;
    },
    getDateOfBirth : function () {
        return this._dateOfBirth;
    }
};
Employee.prototype = {
    setGender : function (gender) {
        this._gender = gender;
    },
    getGender : function () {
        return this._gender;
    }
};
Employee.prototype = {
    setNationality : function (nationality) {
        this._nationality = nationality;
    },
    getNationality : function () {
        return this._nationality;
    }
};
Employee.prototype = {
    setFamilyStatus : function (familyStatus) {
        this._familyStatus = familyStatus;
    },
    getFamilyStatus : function () {
        return this._familyStatus;
    }
};
Employee.prototype = {
    setWebSite : function (webSite) {
        this._webSite = webSite;
    },
    getWebSite : function () {
        return this._webSite;
    }
};
Employee.prototype = {
    setEmail : function (email) {
        this._email = email;
    },
    getEmail : function () {
        return this._email;
    }
};
Employee.prototype = {
    setWorkPlace : function (workPlace) {
        this._workPlace = workPlace;
    },
    getWorkPlace : function () {
        return this._workPlace;
    }
};
Employee.prototype = {
    setAddress : function (address) {
        this._address = address;
    },
    getAddress : function () {
        return this._address;
    }
};
Employee.prototype = {
    setPhoneList : function (phoneList) {
        this._phoneList = phoneList;
    },
    getPhoneList : function () {
        return this._phoneList;
    }
};
Employee.prototype = {
    setAttachmentList : function (attachmentList) {
        this._attachmentList = attachmentList;
    },
    getAttachmentList : function () {
        return this._attachmentList;
    }
};
Employee.prototype = {
    setPhoto : function (photo) {
        this._photo = photo;
    },
    getPhoto : function () {
        return this._photo;
    }
};
